### DISCUSSION TITLE
LDAP groups have no members (Active Directory)

### DISCUSSION THREAD

**Discussion by Thulium-Drake** [kanboard/kanboard#5195] [General]
### Actual behaviour
AD synced groups empty (no members)

* I have configured LDAP with the example from the documentation to allow AD users to log in to kanboard. Works!
* I also enabled the LDAP group provider to allow mapping groups -> project roles. Works!
* When a new AD user logs in, they are created in the kanboard interface, but the groups Kanboard has synced all have 0 users.
### Expected behaviour
Groups to show their members and the permissions granted to those groups applied to the user.

### Steps to reproduce
1. Configure LDAP for users and groups via AD (use docs)
2. Configure a new project with a role for an AD group
3. Log in as AD user with a membership to the in step 2 configured project-group
4. Notice 'You have no projects assigned to you' and Group management shows empty groups.

### Screenshots
![kanboard-nousers](https://user-images.githubusercontent.com/12935477/85132863-f31f0d80-b239-11ea-882e-cc06638230db.png)

### Configuration

- Kanboard version: 1.2.14
- Database type and version: 1:10.3.22-0+deb10u1
- PHP version: 7.3.14-1~deb10u1
- OS: Debian 10
- Browser: Firefox/Chrome

**Comment by MaoX17** [comment]
i've same issue on docker.
is there some workaround?

Versione dell'applicazione: v1.2.15
Versione PHP: 7.3.19
PHP SAPI: fpm-fcgi
HTTP Client: cURL
Versione OS: Linux 3.10.0-1127.19.1.el7.x86_64
Driver per Database mysql
Versione database: 10.5.5-MariaDB-1:10.5.5+maria~focal
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.121 Safari/537.36

**Comment by MaoX17** [comment]
Update:

group sync work.
I have problem with Domain Users maybe becouse i have too many users in it

**Comment by Floyer007** [comment]
@Thulium-Drake  for me the only option was the following: https://github.com/kanboard/kanboard/issues/4312

**Comment by Ice-Drake** [comment]
Did you guys have any luck getting the administrator or manager group to work?
